variation-commons-batch/src/main/java/uk/ac/ebi/eva/commons/batch/job/ManageJobsUtils.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2014-2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.commons.batch.job;
+
+import org.springframework.batch.core.BatchStatus;
+import org.springframework.batch.core.JobExecution;
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.StepExecution;
+import org.springframework.batch.core.repository.JobRepository;
+import org.springframework.util.StringUtils;
+
+
+import uk.ac.ebi.eva.commons.batch.exception.NoJobToExecuteException;
+import uk.ac.ebi.eva.commons.batch.exception.NoParametersHaveBeenPassedException;
+import uk.ac.ebi.eva.commons.batch.exception.NoPreviousJobExecutionException;
+
+import java.util.Date;
+
+/**
+ * Utility class to change job / step status
+ */
+public class ManageJobsUtils {
```

Comment:
`JobUtilsManager` or even `JobStatusManager` (according to class doc) would follow more closely class naming styles. Verbs are not recommended.